### ISSUE TITLE
BUG: (script name)

### ISSUE THREAD

**Issue by alexofthek** [Minesraft2/Blooket-Cheats#970]
### Script

gui

### Problem

hacks do not work anymore whenever you run they do nothing

### Method

Put 'javascript:(code)' in the url bar or a bookmark

### Type

Unobfuscated, Obfuscated

### Extra Info

none

**Comment by DragonDude20** [comment]
youre probs doing it wrong it works just fine
try this link instead

minesraft2.github.io/Blooket-Cheats/#

Then click GUI

Create a new bookmark

name it whatever

then paste (clicking the GUI button copies the code) into the URL bar

much hacks

**Comment by alexofthek** [comment]
its not working for everyone at my school idk
________________________________

**Comment by Minesraft2** [comment]
Your school probably blocked javascript

**Comment by B-Settings** [comment]
> Your school probably blocked javascript

Yes, but if they did, then it wouldn't even show up, so that's probably not the case, it's probably he didn't update his or sum
